lib/helper/storage.js

```diff
@@ -0,0 +1,172 @@
+'use babel';
+
+import { basename, dirname, leadingslashit, trailingslashit, unleadingslashit, untrailingslashit, normalize, cleanJsonString } from './../helper/format.js';
+import { decrypt, encrypt } from './../helper/secure.js';
+import { throwErrorIssue44 } from './../helper/issue.js';
+
+const atom = global.atom;
+const server_config = require('./../config/server-schema.json');
+const folder_config = require('./../config/folder-schema.json');
+
+class Storage {
+
+    constructor() {
+      const self = this;
+
+      self.servers = [];
+      self.folders = [];
+      self.version = '';
+      self.password = null;
+      self.loaded = false;
+    }
+
+    setPassword(password) {
+        const self = this;
+
+        self.password = password;
+    }
+
+    getPassword() {
+        const self = this;
+
+        return self.password;
+    }
+
+    hasPassword() {
+        const self = this;
+
+        return self.password !== null;
+    }
+
+    load(reload = false) {
+        const self = this;
+
+        if(!self.loaded || reload) {
+            let configText = atom.config.get('ftp-remote-edit.config');
+            if (configText) {
+              try {
+                configText = JSON.stringify(JSON.parse(cleanJsonString(configText)), null, 4);
```

Comment:
This can be removed here because the debug configuration has also been removed. This was available in plain text.